Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:2.7-slim
+
+RUN mkdir /web-platform-tests
+WORKDIR /web-platform-tests
+
+ADD . /web-platform-tests/
+RUN echo '{"ports":{"http":[80,"auto"],"https":[443]},"bind_address":false}' > /web-platform-tests/config.json
+
+EXPOSE 80
+EXPOSE 443
+
+CMD [ "python", "wpt", "serve" ]
```

Comment:
I've been using this Dockerfile in the root of the https://github.com/web-platform-tests/wpt project. It is incomplete, we should hardcode additional ports for `ws`, `wss` and a 2nd port instead of `"auto"` for http in the config.json so these ports will not be randomly selected on startup. We will then need to `EXPOSE` these ports with docker.